lib/test.js

```diff
@@ -219,35 +219,59 @@ Test.prototype._assert = function assert (ok, opts) {
         var dir = __dirname + path.sep;
         
         for (var i = 0; i < err.length; i++) {
-            var m = /^[^\s]*\s*\bat\s+(.+)/.exec(err[i]);
+            /*
+                Stack trace lines may resemble one of the following. We need
+                to should correctly extract a function name (if any) and
+                path / line no. for each line.
+            
+                at myFunction (/path/to/file.js:123:45)
+                at myFunction (/path/to/file.other-ext:123:45)
+                at myFunction (/path to/file.js:123:45)
+                at myFunction (C:\path\to\file.js:123:45)
+                at myFunction (/path/to/file.js:123)
+                at Test.<anonymous> (/path/to/file.js:123:45)
+                at Test.bound [as run] (/path/to/file.js:123:45)
+                at /path/to/file.js:123:45
+            */
+            var m = new RegExp(
```

Comment:
it would be really nice if we could use a regex literal, and repeat the parts you want to comment, in the comment, instead of using `new RegExp`